ethereum/blockwatch/block_watcher.go

```diff
@@ -66,6 +67,7 @@ func (e TooMayBlocksBehindError) Error() string {
 
 // Config holds some configuration options for an instance of BlockWatcher.
 type Config struct {
+	MaxMiniHeaders  int
```

Comment:
It looks like we never actually use this?

Reply:
Yikes, I probably needed to give this PR a once over before putting it up for review. I'll make sure I do that in the future